### DISCUSSION TITLE
Auto-linkify URLs in code like on github.com

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#9758] [Other Feature Feedback, Questions, & Ideas] [answered]
Currently, the URLs are not linkified on cs.github.com, while they are on github.com:
- https://cs.github.com/cli/cli/blob/eaa64df80193f4f632d574f25729460d8b657819/internal/authflow/flow.go?q=legacy+callback+URL#L78
- https://github.com/cli/cli/blob/eaa64df80193f4f632d574f25729460d8b657819/internal/authflow/flow.go#L78

**Comment by look** [comment, marked as answer]
Hi @jack1142, at least for me, the GitHub.com code viewer doesn't make URLs clickable. Do you have a browser extension that does this, perhaps?

**Reply by ZeWaka** [reply]
I have an extension that does so, so I presume it's the same for the OP.

**Reply by Jackenmen** [reply]
> Do you have a browser extension that does this, perhaps?

Yep, my bad! I forgot to first check that this isn't an extension feature.

For reference, it's the [`linkify-code` feature from Refined GitHub extension](https://github.com/refined-github/refined-github/blob/main/source/features/linkify-code.tsx).
